### PR TITLE
check-pause-release-suspend-resume: small fixes

### PR DESCRIPTION
**Pull request by marc-hb**
3 commits, check messages.

Note this test seems orphaned, it was notably broken by commit
https://github.com/thesofproject/sof-test/commit/46dadd0c563e6caa0c01c4acec6caf4bdfc30d7f ("Add mutual exclusion and journalctl logging") and no one
noticed (I temporarily disabled https://github.com/thesofproject/sof-test/commit/46dadd0c563e6caa0c01c4acec6caf4bdfc30d7f locally)

### CODE REVIEW

**Comment by marc-hb** [comment]
only 1 shellcheck warning left, it's in expect code so very hard to fix

File: test-case/check-pause-release-suspend-resume.sh [hunk 1]
```diff
@@ -51,7 +51,10 @@ set -e
 ##  * No unexpected errors should be present in dmesg during or after test
 ##      completion.
 
-source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+TOPDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")"/..; pwd)
+
+# shellcheck source=case-lib/lib.sh
+source "$TOPDIR"/case-lib/lib.sh
```
Comment (paulstelian97) [inline test-case/check-pause-release-suspend-resume.sh:57]: Wait, this wasn't already done?

**Review by paulstelian97 — APPROVED** [review]

File: test-case/check-pause-release-suspend-resume.sh (hunk 1 above)
Comment (marc-hb) [reply test-case/check-pause-release-suspend-resume.sh:57]: It was on line 54? Not sure I understand the question sorry.

**Comment by marc-hb** [comment]
Thanks for the review!

Minor fixes in an unused test that is still broken - merging. Will reduce some shellcheck noise in #931

**Pull request merged**